t/venom/test_suites/captive_portal/25_check_radius_audit_log.yml

```diff
@@ -68,16 +75,6 @@ testcases:
                   "value": "{{.captive_portal.profiles.filters.connection_type}}"
                 }
               ]
-            },            
-            {
-              "op": "or",
-              "values": [            
-                {
-                  "field": "created_at",
-                  "op": "greater_than",
-                  "value": "{{.get_time.two_minutes_ago}}"
-                }
-              ]
```

Comment:
This as been remove since multiple retry, correct?
Then this part is not useful anymore.

Same for the other

```
- name: get_time
  steps:
  - type: exec
    script: "date '+%Y-%m-%d %H:%M:%S' --date='2 minutes ago'"
    vars:
      two_minutes_ago:
        from: result.systemout
```